### ISSUE TITLE
multiboxing via Steam client

### ISSUE THREAD

**Issue by smthnghppns** [Healix/Gw2Launcher#388]
Hey, is it possible to run multiboxing via steam accounts (i got 4 steam accounts)? or i should use sandbox for steam?

**Comment by Healix** [comment]
No. I'm assuming GW2 periodically communicates with Steam, so each account would require its own instance of Steam. If it doesn't, and it's possible to simply switch Steam accounts and launch GW2 again, then it would be possible (if I removed the limiter).

**Comment by smthnghppns** [comment]
> No. I'm assuming GW2 periodically communicates with Steam, so each account would require its own instance of Steam. If it doesn't, and it's possible to simply switch Steam accounts and launch GW2 again, then it would be possible (if I removed the limiter).
so only one way, just use sandboxie or simuliar soft.
but gw2launcher cant help me with that(
